.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,17 +1,17 @@
 ---
 name: Feature request
 about: Suggest an idea for this project
-
+labels: enhancement
```

Comment:
Would it make sense to add the "not ready for work" label here so we can triage?